relationshiptypes.md

```diff
@@ -68,3 +68,71 @@ A sample can be found at https://dspace7-entities.atmire.com/rest/#https://dspac
 ```
 
 The 2 [item types](itemtypes.md) are embedded
+
+## Search methods
+
+### Relationship types containing an entity type
+**/api/core/relationshiptypes/search/byEntityTypeId?id=<:entity-type-id>**
+
+Parameters:
+* The `id` should be the entity type id from the [entity types endpoint](entitytypes.md). It is mandatory. It can occur on either the left or right hand side
+
+A sample search would be /server/api/core/relationshiptypes/search/byEntityTypeId?id=1
+
+It would respond with
+```json
+{
+  "_embedded": {
+    "relationshiptypes": [
+      {
+        "id": 10,
+        "leftwardType": "isAuthorOfPublication",
+        "rightwardType": "isPublicationOfAuthor",
+        "copyToLeft": false,
+        "copyToRight": false,
+        "leftMinCardinality": 0,
+        "leftMaxCardinality": null,
+        "rightMinCardinality": 0,
+        "rightMaxCardinality": null,
+        "type": "relationshiptype"
+      },
+      {
+        "id": 1,
+        "leftwardType": "isAuthorOfPublication",
+        "rightwardType": "isPublicationOfAuthor",
+        "copyToLeft": false,
+        "copyToRight": false,
+        "leftMinCardinality": 0,
+        "leftMaxCardinality": null,
+        "rightMinCardinality": 0,
+        "rightMaxCardinality": null,
+        "type": "relationshiptype"
+      }
+    ]
+  }
+}
+```
+
+## Property-based projections
+
+[Property-based projections](projections.md#property-based-projections) can add new JSON properties to the response. When requesting the projection, any `relationshiptype` in the response will add these properties.  
+The REST request doesn't have to be on the `relationshiptype` endpoint directly, it can be on another endpoint which simply embeds relationship types.
+
+### Verify whether a given entity type is the left or right
+**?projection=CheckSideEntityInRelationshipType&checkSideEntityInRelationshipType=<:entitytype>**
```

Comment:
Also might suggest simplifying the name here to something like `CheckRelatedTypeSide` or `CheckSideOfRelatedType`...based on the name of the projection you suggest for the relationships endpoint.

Reply:
I agree that simplifies the name if you know in advance this is specific to a RelationshipType.
Would you want to keep the name, or do you expect the name you suggested will be clear enough to indicate it adds the information to a RelationshipType?